python/whylogs/api/logger/result_set.py

```diff
@@ -183,10 +178,6 @@ def reader(name: str = "local") -> "ResultSetReader":
         reader = Readers.get(name)
         return ResultSetReader(reader=reader)
 
-    def writer(self, name: str = "local") -> "ResultSetWriter":
```

Comment:
I moved `.writer()` into the `Writable` base class and made a generic `WriterWrapper` so that we don't need a separate `*Writer` for every `Writable` subclass. This should be backwards compatible.